### DISCUSSION TITLE
Poem as reversed proxy

### DISCUSSION THREAD

**Discussion by phungleson** [poem-web/poem#131] [General]
I am looking for some sort of reversed proxy solution and not sure if poem is a suitable solution for this.

Basically I am looking for a way to authenticate users and if users satisfied certain condition I will proxy the request to different services like an aws lambda somewhere.

**Comment by sunli829** [comment]
I'll update the `poem-gateway` dependency to the latest version of `poem`, you should be able to get inspiration from it. 🙂
